sdk/python/lib/pulumi/output.py

```diff
@@ -213,7 +213,7 @@ def apply(self, func: Callable[[T], Input[U]], run_with_unknowns: Optional[bool]
         run_fut = asyncio.ensure_future(run())
         return Output(result_resources, run_fut, result_is_known, result_is_secret)
 
-    def __getattr__(self, item: str) -> 'Output[Any]':
+    def __getattr__(self, item: str) -> 'Output[Any]': # type: ignore
```

Comment:
Here we're failing the `is_subtype` check at the [end](https://github.com/python/mypy/blob/6e567a655a0d13f5d2cc3bc9531d06a0e5bd0c5b/mypy/checker.py#L1348-L1372).

[Mypy docs](https://mypy.readthedocs.io/en/latest/common_issues.html) recommend annotating __getattr__ calls with `Any` return types, but I think this is more user-friendly.

Reply:
what does this mean?  do callers ignore the type?  or does it mean that the type checker ignores within this method?